openfl/federated/task/runner_keras.py

```diff
@@ -17,8 +18,8 @@
 
 with catch_warnings():
     simplefilter(action="ignore")
+    import keras as ke
     import tensorflow as tf
```

Comment:
Ideally tensorflow must not be used with Keras as Keras implementation must be free of any backend while running.
Please find the alternative for the same.

Reply:
tensorflow is removed.